docs/markdown/snippets/build_target_deprecate_jar.md

```diff
@@ -0,0 +1,6 @@
+## passing `target_type : 'jar'` to `build_target()` is deprecated
+
+This never really made sense, as you can't mix java sources with non-java
```

Comment:
Nitpick, but capitalize the j in Java?

Reply:
The comma after sense could probably be removed

Reply:
done